### PR TITLE
Added 1.15 support and changed code format

### PR DESCRIPTION
**Pull request by NgLoader**
- 1.15 Support
- Changed code format
- Added github CI for automaticly build and releases
- Changed README.md

To release a new version use:
1. git tag <version>
2. git push origin <version>

### CODE REVIEW

**Comment by NgLoader** [comment]
More clean up and better project structuring coming soon

**Comment by RoboMWM** [comment]
This is a maven project...

File: README.md [hunk 1]
```diff
@@ -1,58 +1,78 @@
-# GriefPrevention - Minecraft anti-griefing plugin
+<img align="right" src="https://media.forgecdn.net/avatars/73/181/636163527796328705.png" height="200" width="200">
 
-Stop responding to grief and prevent it instead. Grief Prevention will solve
-your grief problems without a roster of trained administrators, without 10
-different anti-grief plugins, and without disabling any standard game features.
-Because Grief Prevention teaches players for you, you won't have to publish a
-training manual or tutorial for players, or add explanatory signs to your world.
+# GriefPrevention - Minecraft anti-griefing plugin [![Build Status](https://github.com/TeamWuffy/GriefPrevention/workflows/Build/badge.svg)](https://github.com/TeamWuffy/GriefPrevention/actions?query=workflow%3ABuild) [![Release Status](https://github.com/TeamWuffy/GriefPrevention/workflows/Releases/badge.svg)](https://github.com/TeamWuffy/GriefPrevention/releases/latest)
```
Comment (RoboMWM) [inline README.md:3]: Appreciate the PR but this diff is way too big and is apparently customized for your fork. Please consider breaking up this PR into smaller PRs that each address a single issue.

File: pom.xml [hunk 2]
```diff
@@ -1,119 +1,84 @@
...
+		</repository>
+		<repository>
+			<id>jitpack.io</id>
+			<url>https://jitpack.io</url>
+		</repository>
+	</repositories>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.spigotmc</groupId>
```
Comment (RoboMWM) [inline pom.xml:35]: GP uses CB as the dependency, not spigot. (Yes, there are a fair bit of servers on CB.)

File: pom.xml [hunk 3]
```diff
@@ -1,119 +1,84 @@
...
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<version>3.2.1</version>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
```
Comment (RoboMWM) [inline pom.xml:77]: Why are we shading?

**Comment by NgLoader** [comment]
I see, my error sorry I commit some changes

**Review by RoboMWM — CHANGES_REQUESTED** [review]
Appreciate the PR but this diff is _way_ too big, especially since you mixed in code reformatting in, making the diff practically unreadable. Please consider breaking up this PR into smaller PRs that each address a single issue.

For tips and guidelines, see the [contributing.md](https://github.com/TechFortress/GriefPrevention/blob/master/.github/CONTRIBUTING.md)

File: README.md (hunk 1 above)
Comment (NgLoader) [reply README.md:3]: okay

File: pom.xml (hunk 2 above)
Comment (NgLoader) [reply pom.xml:35]: I already changed it on my local branch

File: pom.xml (hunk 3 above)
Comment (NgLoader) [reply pom.xml:77]: I changed to your pom.xml ignore this pls

**Pull request closed without merging**